### ISSUE TITLE
Missing `info` field in `VcfRecord`

### ISSUE THREAD

**Issue by yaseminbridges** [phenopackets/phenopacket-tools#184]
Hi,

I am using phenopacket-tools-1.0.0-RC3-SNAPSHOT to convert a v1 phenopacket to a v2 phenopacket. The `--convert-variants` appear to miss translating the `info` in the v1 phenopacket to the v2 phenopacket. i.e., in the v1 phenopacket:

```json
  "variants": [{
    "vcfAllele": {
      "genomeAssembly": "GRCh38",
      "id": "68f0e032ddc263583177b032d3b7d6cf",
      "chr": "5",
      "pos": 112699127,
      "ref": "N",
      "alt": "DEL",
      "info": "SVTYPE=DEL;END=112710148"
    },
    "zygosity": {
      "id": "GENO:0000135",
      "label": "heterozygous"
    }
  }],
```

when converting the output is:

```json
          "variationDescriptor": {
            "id": "68f0e032ddc263583177b032d3b7d6cf",
            "vcfRecord": {
              "genomeAssembly": "GRCh38",
              "chrom": "5",
              "pos": "112699127",
              "ref": "N",
              "alt": "DEL"
            },
            "moleculeContext": "genomic",
            "allelicState": {
              "id": "GENO:0000135",
              "label": "heterozygous"
            }
```

Thanks,

Yasemin

**Comment by ielis** [comment]
Hi @yaseminbridges
thanks a lot for reporting the bug! I fixed it in #185  and the updated code is available in the *latest* code (as described by the top-level README.md) which you can get when you build from source. 

I suspect you know how to do it but just to be sure:
```shell
cd phenopacket-tools
git checkout develop
git pull
./mvnw -Prelease clean package
```
This will create a distribution ZIP in `phenopacket-tools-cli/target` folder.

Please let me know if the fix did not work. Otherwise, I'll close this issue in a few days.

**Comment by yaseminbridges** [comment]
Hi @ielis,

Just tested the latest code and it works great, thanks!
